### PR TITLE
Sbm 1305

### PR DESCRIPTION
**Pull request by RomanCRS**
This cookbook enables VPC ClassicLink on EC2 Classic instances.

Related: easybib/sbm-issues#1305

### CODE REVIEW

File: vpc-classiclink/recipes/classiclink.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#
+# Cookbook Name:: vpc-classiclink
+# Recipe:: classiclink
+#
+
+bash "attach-classiclink-to" do
```
Comment (till) [inline vpc-classiclink/recipes/classiclink.rb]: use `execute` instead.

File: vpc-classiclink/recipes/install.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#
+# Cookbook Name:: vpc-classiclink
+# Recipe:: install
+#
+include_recipe 'python::pip'
+
+python_pip "awscli" do
```
Comment (till) [inline vpc-classiclink/recipes/install.rb]: Please re-use the existing cookbook:
https://github.com/till/easybib-cookbooks/tree/master/awscli

File: vpc-classiclink/templates/default/aws.credentials.erb [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[default]
+aws_access_key_id = <%=node["vpc-classiclink"]["classiclink_iam_id"] %>
```
Comment (till) [inline vpc-classiclink/templates/default/aws.credentials.erb]: Not sure what you think, but I would prefer using an instance policy so we don't have to manage these keys at all.

**Comment by RomanCRS** [comment]
@till I was able to test cookbook on my test OpsWorks stack. At this moment it's not working properly because we need to specify a region. Also, we need to use the following version of AWS CLI - 1.11.80
Can you replace execution command with the following and add attribute "region"?
```
aws ec2 attach-classic-link-vpc --instance-id #{instance_id} --vpc-id #{vpc_id} --groups #{sg} --region #{region}
```

**Comment by till** [comment]
@RomanCRS 

You can set the version in the attributes:

```
{
  "awscli": {
    "version": "1.11.80"
  }
}
```

I am adding region.

**Comment by RomanCRS** [comment]
@till I've set this attribute during my testing. I just wrote this to let you know that we need this in each stack where we want to use this cookbook.

**Comment by till** [comment]
@RomanCRS okay, i pushed up a fix for the missing region. Let me know. If this works, then let's do a version upgrade with another PR.

**Comment by RomanCRS** [comment]
@till Cookbook is working fine now. I've updated custom JSON of API Staging and API OpsWorks stacks with proper attributes. We can apply cookbook to API Staging now.

**Comment by RomanCRS** [comment]
@till I've created a new instance in API Staging OpsWorks stack. This instance has been linked to proper VPC automatically. Also, I've been able to connect to DB from that instance:
```
rchyr@nginxphpapp2:~$ telnet cm-bibme-production-2.cln9n5lljjdz.us-east-1.rds.amazonaws.com 3306
Trying 172.31.37.179...
Connected to ec2-52-7-205-194.compute-1.amazonaws.com.
Escape character is '^]'.
N
5.6.34-log�Co|CxV"Ck�t#PGUV0*"injmysql_native_password^CConnection closed by foreign host.
```
Would you like to proceed with production?

**Comment by till** [comment]
@RomanCRS I made #1220 — but we have to update easybib production PHP with this PR. Since it's been running for a while, it should be fine. But we basically need to relaunch all `nginxphpapp` in "EasyBib" stack.

**Comment by till** [comment]
(As far as I read the diff. I haven't checked too much.)

**Comment by RomanCRS** [comment]
@till If you want we can merge PR tomorrow and I'll re-create instances in Easybib stack tomorrow one by one. Is it suitable for you?

**Comment by till** [comment]
@RomanCRS yeah, ping me in the morning. We'll get this sorted.

**Pull request merged**